pandas/core/series.py

```diff
@@ -398,6 +398,12 @@ def _values(self):
         """ return the internal repr of this data """
         return self._data.internal_values()
 
+    def _formatting_values(self):
+        """Return the values that can be formatted (used by Series and
+        DataFrameFormatter)
+        """
+        return self._data._block.formatting_values()
```

Comment:
@jreback I could also add one extra level of redirection. Instead of now Series -> Block, I could also do Series -> SingleBlockManager -> Block. For that I just need to add a `formatting_values` to SingleBlockManager that calls Block.formatting_values(). But is seems an extra unneeded layer, although for `internal_values` or `get_values` it is done like this.